python/pyarrow/_dataset.pyx

```diff
@@ -1013,27 +1013,36 @@ cdef class ParquetReadOptions(_Weakrefable):
     dictionary_columns : list of string, default None
         Names of columns which should be dictionary encoded as
         they are read.
+    enable_parallel_column_conversion : bool, default False
+        Whether single files may be read in parallel (ignored when reading
+        multiple files to avoid thread contention).
```

Comment:
Add note that this is experimental. Once we fix our nested parallelism problem this shouldn't be needed

Reply:
alright